src/services/info/notice/notice-update/NoticeUpdatePage.vue

```diff
@@ -56,6 +50,7 @@ export default {
                 });
             } catch (e) {
                 ErrorHandler.handleError(e);
+                state.noticePostData = {} as NoticePostModel;
```

Comment:
위 state.noticePostData 선언부에 제안드린 방식으로 진행하면 아래처럼 바뀌어도 괜찮을 것 같습니다.
```suggestion
                state.noticePostData = {};
```